src/index.js

```diff
@@ -0,0 +1,5 @@
+export default function foo() {
+  return 3;
+}
+
+// module.exports = foo;
```

Comment:
It has no impact on the functionalities of your code, but it having old and commented code makes your code less readable. So, it is advised to remove old codes 😀